CHANGELOG.asciidoc

```diff
@@ -12,7 +12,7 @@ endif::[]
 
 [float]
 ===== Features
-* Cool new feature: {pull}2526[#2526]
+- Make TRANSACTION_IGNORE_URLS dynamically configurable: {pull}872[#872]
```

Comment:
The changelog entry should be lower down, under "View commits" on line 24. We don't currently split up between fixes, features, etc.